docs/editor/vscode-web.md

```diff
@@ -17,67 +17,77 @@ VS Code for the Web runs entirely in your web browser, so there are certain limi
 
 ## Relationship to VS Code Desktop
 
-VS Code for the Web provides a browser-based experience for navigating files and repositories and committing lightweight code changes. However, if you need access to a runtime to run, build, or debug your code, or you want to use platform features such as a terminal, we recommend moving your work to the desktop application or [GitHub Codespaces](https://github.com/features/codespaces) for the full capabilities of VS Code. In addition, VS Code Desktop lets you run extensions that aren't supported in the web version, and use a full set of keyboard shortcuts not limited by your browser.
+VS Code for the Web provides a browser-based experience for navigating files and repositories and committing lightweight code changes. However, if you need access to a runtime to run, build, or debug your code, you want to use platform features such as a terminal, or you want to run extensions that aren't supported in the web, we recommend moving your work to the desktop application, [GitHub Codespaces](https://github.com/features/codespaces), or using [Remote-Tunnels](#use-your-own-compute-with-remote-tunnels) for the full capabilities of VS Code. In addition, VS Code Desktop lets you use a full set of keyboard shortcuts not limited by your browser.
```

Comment:
It was right before -- the name is "[Remote - Tunnels](https://marketplace.visualstudio.com/items?itemName=ms-vscode.remote-server)" and that's how we refer it to and other extensions (e.g. [Remote - SSH](https://code.visualstudio.com/docs/remote/ssh-tutorial))

Reply:
Yeah we aren't always the most consistent with it, so I was open to making it "Remote-Tunnels" if folks found that more readable. But I think you're right we should just stick with the official spacing. I'll make that change back.

Reply:
Updated here and in a variety of other spots in this doc.